phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/MainController.java

```diff
@@ -30,13 +30,25 @@
 import org.phoenicis.javafx.settings.JavaFxSettingsManager;
 import org.phoenicis.javafx.views.common.ThemeManager;
 import org.phoenicis.javafx.views.mainwindow.ui.MainWindow;
+import org.phoenicis.repository.RepositoryManager;
+import org.phoenicis.repository.dto.CategoryDTO;
+import org.phoenicis.repository.dto.RepositoryDTO;
+import org.slf4j.LoggerFactory;
 
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
 import java.util.Optional;
 
 import static org.phoenicis.configuration.localisation.Localisation.tr;
 
 public class MainController {
+    private final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(MainController.class);
```

Comment:
please import the `Logger` class in the header.

Reply:
You also need to change the line to:
```java
private final Logger LOGGER = LoggerFactory.getLogger(MainController.class);
```

Reply:
Makes sense. Copy+paste...